Expand documentation, Python 3 support

RoelN opened the pull request:
I added some documentation as I couldn't get it up and running without these steps. Also quickly moved everything to Python 3.

RoelN committed:
Update to Python 3, formatting

RoelN committed:
Update readme, add more missing dependencies, rename file

davelab6 commented on pdf2slides.py:
```diff
@@ -0,0 +1,76 @@
+    print("Syntax: " + sys.argv[0] + " <pdf file>")
+    sys.exit()
+
+pages = Image(filename=sys.argv[1], resolution=300)
```
Suggest to make resolution `argv[2]`

davelab6 commented on pdf2slides.py:
```diff
@@ -0,0 +1,76 @@
+print("Removed blank slide")
+
+# Populate the slides
+i = 0
```
Maybe should start from 1?

RoelN replied:
It's only for output during build, so doesn't affect created Slides document, but changed anyway :-)

RoelN committed:
Take resolution as argument

Continued using sys.argv as it's relativelty simple. Use argparse
if more arguments.

RoelN commented:
@elliotkendall I hope you appreciate these tweaks, if you have any questions, please let me know :-)

elliotkendall commented:
Thanks, folks! I'm glad you found this code useful
